macros/optimize_tables.sql

```diff
@@ -167,6 +167,9 @@ OPTIMIZE sudoswap_ethereum.pool_trades;
 OPTIMIZE airswap_ethereum.trades;
 {% endset %}
 
+{% set dodo_ethereum_trades %}
+OPTIMIZE dodo_ethereum_trades;
```

Comment:
Here may be `dodo_ethereum.trades`, but this file has been changed in the latest version of main branch.